setup.py

```diff
@@ -144,6 +144,7 @@ def find_packages(path, base=""):
     "mediajson>=1.0.0",
     "authlib>=0.10",
     "pyopenssl",
+    "cryptography",
```

Comment:
I'm slightly surprised this wasn't here already. Is this likely to cause any issues for @bbc/rd-apmm-cloudfit?

Reply:
authlib already depends on cryptography.. I added it explicitly due to the changes in this branch now calling cryptography directly